js/src/product-feed/submission-success-guide/enhanced-conversions/confirm-button.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ENHANCED_ADS_CONVERSION_STATUS } from '.~/constants';
+import { useAppDispatch } from '.~/data';
+import AppButton from '.~/components/app-button';
+
+const ConfirmButton = ( { onConfirm = noop } ) => {
+	const { updateEnhancedAdsConversionStatus } = useAppDispatch();
+
+	const handleConfirm = () => {
```

Comment:
We're going to need to move this out to the CTA component and pass it in as part of the `onConfirm` callback that is passed to this component because the requirements have changed slightly, and we want to automatically confirm that this is enabled once the user has clicked from the T&C modal and returned, without needing to take a second step of also confirming again. This should also work in the settings panel if the user clicks the `EnableButton` and then returns to the settings page.

Reply:
👍